src/kernel/include/cap_ops.h

```diff
@@ -4,6 +4,7 @@
 #include "captbl.h"
 #include "pgtbl.h"
 #include "liveness_tbl.h"
+#include "chal/chal_prot.h"
```

Comment:
What's `prot` for? I was thinking it meant `prototype`, in which case at least `proto` for short?